rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-release-service-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: release_service_alerts
+    interval: 1m
+
+    rules:
+    # record rules for `ReleaseServiceProcessingDuration`
+    - record: rate_of_release_processing_duration_seconds_longer_than_1h_per_10m
+      expr: sum by (job) (rate(release_processing_duration_seconds_bucket{le="3600"}[10m]))
+
+    - record: rate_of_release_processing_duration_seconds_counter_per_10m
+      expr: sum by (job) (rate(release_processing_duration_seconds_count[10m]) > 0)
```

Comment:
What's the reason for adding the `> 10` on those rules?

Reply:
I am not sure if this is a feature or a bug but if the `>0` is not added, in case we have no occurrences in the desired bucket the function returns no data (maybe due to the division by 0). By adding it only calculates if there is data available, hence when there is no data, the value is 1.00.

Reply:
alright then.